app/src/main/java/org/eyeseetea/malariacare/layout/adapters/dashboard/AAssessmentAdapter.java

```diff
@@ -62,73 +64,26 @@ public View getView(int position, View convertView, ViewGroup parent) {
-            Date completionDate = survey.getCompletionDate();
-            SimpleDateFormat format = new SimpleDateFormat("MM/dd/yyyy");
-            sentDate.setText(format.format(completionDate));
+        //Completion Date
+        TextCard completionDate = (TextCard) rowView.findViewById(R.id.completionDate);
+        if(survey.getCompletionDate()!=null){
+            DateFormat formatter=DateFormat.getDateInstance(DateFormat.SHORT, Resources.getSystem().getConfiguration().locale);
+            completionDate.setText(formatter.format(survey.getCompletionDate()));
         }
 
-        // show facility name (or not) and write survey type name
-        if (!showNextFacilityName) {
-            facilityName.setVisibility(View.GONE);
-            facilityName.setLayoutParams(new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, 0, 0f));
-            surveyType.setLayoutParams(new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, 0, 1f));
-        } else {
-            facilityName.setText(survey.getOrgUnit().getName());
-            facilityName.setLayoutParams(new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, 0, 0.5f));
-            surveyType.setLayoutParams(new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, 0, 0.5f));
-        }
-        surveyType.setText("- " + survey.getProgram().getName());
-
-        // check whether the following item belongs to the same org unit (to group the data related
-        // to same org unit with the same background)
-        if (position < (this.items.size()-1)) {
-            if (this.items.get(position+1).getOrgUnit().equals((this.items.get(position)).getOrgUnit())){
-                // show background without border and tell the system that next survey belongs to the same org unit, so its name doesn't need to be shown
-                rowView.setBackgroundResource(LayoutUtils.calculateBackgroundsNoBorder(this.backIndex));
-                this.showNextFacilityName = false;
-            } else {
-                // show background with border and switch background for the next row
-                rowView.setBackgroundResource(LayoutUtils.calculateBackgrounds(this.backIndex));
-                this.backIndex++;
-                this.showNextFacilityName = true;
-            }
-        }  else {
-            //show background with border
-            rowView.setBackgroundResource(LayoutUtils.calculateBackgrounds(this.backIndex));
-        }
+        //RDT
+        TextCard rdt = (TextCard) rowView.findViewById(R.id.rdt);
+        rdt.setText(survey.isRDT()?"+":"-");
 
-        //Status Cell
-        ((TextCard) rowView.findViewById(R.id.score)).setText(getStatus(survey));
+        //INFO
+        TextCard info = (TextCard) rowView.findViewById(R.id.info);
+        info.setText(survey.getValuesToString());
 
+        rowView.setBackgroundResource(LayoutUtils.calculateBackgrounds(position));
         return rowView;
     }
 
-    /**
```

Comment:
@ivanarrizabalaga We are going to have sth like a progress bar (2 of 7) maybe it is a good idea to not delete this method and just change the implementation. However you know the code better than me so does it make sense?